tests/foreman/api/test_partitiontable.py

```diff
@@ -101,6 +101,56 @@ def test_positive_create_with_os(self):
         ptable = entities.PartitionTable(os_family=os_family).create()
         self.assertEqual(ptable.os_family, os_family)
 
+    def test_positive_create_with_org(self):
+        """Create new partition table with organization
+
+        :id: 5f97b180-3708-4e1c-8407-42977459d4b6
+
+        :expectedresults: Partition table created successfully and has correct
+            organization assigned
+
+        :CaseImportance: Critical
+        """
+        org = entities.Organization().create()
+        ptable = entities.PartitionTable(organization=[org]).create()
+        self.assertEqual(ptable.organization[0].read().name, org.name)
+
+    def test_positive_search(self):
+        """Create new partition table and try to find it using its name
+
+        :id: 08520746-444b-47c9-a8a3-438170147453
+
+        :expectedresults: Search functionality works as expected and correct
+            partition table returned
+
+        :CaseImportance: Critical
+        """
+        ptable = entities.PartitionTable().create()
+        result = entities.PartitionTable().search(
+            query={'search': ptable.name})
+        self.assertEqual(result[0].id, ptable.id)
```

Comment:
Please also add assertion for len() of result to make sure searching actually works and haven't returned any extra ptables.